protocol/x/clob/keeper/process_operations.go

```diff
@@ -136,6 +136,13 @@ func (k Keeper) ProcessInternalOperations(
 	// All short term orders in this map have passed validation.
 	placedShortTermOrders := make(map[types.OrderId]types.Order, 0)
 
+	affiliatesWhitelistMap, err := k.affiliatesKeeper.GetAffiliateWhitelistMap(ctx)
```

Comment:
Can you do it in a way that make this state read at most once when there are matches, but zero if there's no match: declare `affiliatesWhitelistMap` outside of the scope of for loop, but assign to it only if it's not been initialized within the loop